### DISCUSSION TITLE
what is axum?

### DISCUSSION THREAD

**Discussion by 799189288** [tokio-rs/axum#2103] [General]
Why is it named axum? Is there any special meaning？
Previously, I thought axum was the name of a god of war, one war five,
But there is really very little relevant information online.

**Comment by davidpdrsn** [comment]
Its named after https://en.wikipedia.org/wiki/Axum

**Reply by 799189288** [reply]
so axum is a town in the Tigray Region of Ethiopia with a population of 66,900 residents (as of 2015).

**Reply by snackshell** [reply]
yes it is. i'm from axum.
